.github/workflows/version-bump.yml

```diff
@@ -85,13 +85,14 @@ jobs:
           yarn version --new-version $NEW_VERSION --no-git-tag-version
   
       #
-      # Docs (after alpha)
+      # Docs (after alpha) 
+      # TODO: fix and uncomment
       # 
-      - name: Update docs (for major, minor or patch release)
-        working-directory: ./docs
-        if: ${{ env.bump-type == 'major' || env.bump-type == 'minor' || env.bump-type == 'patch'}}
-        run: |
-          ./build-docs.sh $NEW_VERSION true
+      # - name: Update docs (for major, minor or patch release)
+      #   working-directory: ./docs
+      #   if: ${{ env.bump-type == 'major' || env.bump-type == 'minor' || env.bump-type == 'patch'}}
+      #   run: |
+      #     ./build-docs.sh $NEW_VERSION true
```

Comment:
Hey @samredai and @shangyian . This command fails because there is a 3rd argument missing for BASE_URL, but this never ran for the `alpha` releases so I temporarily commented it out. I am not sure what's the best fix here, but I think I am missing how we updated the docs until now.

Reply:
Agree. I deleted this section.